bin/ungit

```diff
@@ -11,19 +10,10 @@ var BugTracker = require('../src/bugtracker');
 var bugtracker = new BugTracker('launcher');
 var usageStatistics = require('../src/usage-statistics');
 
-var child = new (forever.Monitor)(path.join(__dirname, '..', 'src', 'server.js'), {
-  silent: false,
-  minUptime: 2000,
-  max: config.maxNAutoRestartOnCrash,
-  cwd: path.join(process.cwd(), '..'),
-  options: process.argv.slice(2),
-  env: { LANG: 'en_US.UTF-8' }
-});
-
-process.on('exit', function() {
-  child.kill();
-  console.log('Exit: killed children')
-});
+var child = child_process.spawn('node',
+  [path.join(__dirname, '..', 'src', 'server.js')].concat(process.argv.slice(2)),
+  { cwd: path.join(process.cwd(), '..') }
```

Comment:
What about the `env` setting?

Reply:
It didn't seems to be respected as apparent by #959.  I may leave it in with a note though...